Electrify.SmartMeterUi/Components/Pages/SmartMeterHome.razor.cs

```diff
@@ -68,6 +75,41 @@ private void UpdateDial()
         StateHasChanged();
     }
 
+    private void CheckForNewToastMessage()
+    {
+        bool connected = ErrorMessageService.IsConnected;
+        string? errorMessage = ErrorMessageService.ErrorMessage;
+
+        static void toastOptions(ToastSettings options)
+        {
+            options.DisableTimeout = true;
+            options.ShowCloseButton = false;
+        }
+
+        if (connected)
+        {
+            ToastService.ShowError(_disconnectMessage, toastOptions);
```

Comment:
idk if its wrong or im misunderstanding, but why would you want to show the error message if its connected?

Reply:
because im dumb

Reply:
fixed